crates/rattler_conda_types/src/version/bump.rs

```diff
@@ -335,3 +367,54 @@ mod test {
         );
     }
 }
+
+#[cfg(test)]
+mod semver_tests {
+
+    use std::str::FromStr;
+
+    use crate::{Version, VersionBumpType};
+
+    #[test]
+    fn test_semantic_version_basic() {
```

Comment:
I think you could merge these tests with the existing ones as well by adding them as cases.